tests/release/service/release_plan_and_admission_matched.go

```diff
@@ -19,43 +20,59 @@ var _ = framework.ReleaseServiceSuiteDescribe("ReleasePlan and ReleasePlanAdmiss
 	defer GinkgoRecover()
 
 	var fw *framework.Framework
+	var kubeAdminClient *framework.ControllerHub
 	var err error
-	var devNamespace string
-	var managedNamespace = utils.GetGeneratedNamespace("plan-and-admission-managed")
+	var devNamespace = "rel-plan-admis"
+	var managedNamespace = "plan-and-admission-managed"
 
 	var releasePlanCR, secondReleasePlanCR *releaseApi.ReleasePlan
 	var releasePlanAdmissionCR *releaseApi.ReleasePlanAdmission
+
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
+
 	AfterEach(framework.ReportFailure(&fw))
 
 	BeforeAll(func() {
-		// Initialize the tests controllers
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("rel-plan-admis"))
-		Expect(err).NotTo(HaveOccurred())
-		devNamespace = fw.UserNamespace
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
+			managedNamespace = utils.GetGeneratedNamespace(managedNamespace)
```

Comment:
Below sentence need to be added in this block?
`kubeAdminClient = fw.AsKubeAdmin`